api-channel-issue-tracker/src/main/java/com/blackduck/integration/alert/api/channel/issue/tracker/search/IssueTrackerSearcher.java

```diff
@@ -109,39 +114,42 @@ private List<ActionableIssueSearchResult<T>> findIssuesByAllComponents(ProviderD
             .collect(Collectors.toList());
     }
 
-    private ActionableIssueSearchResult<T> findIssueByProjectIssueModel(ProjectIssueModel projectIssueModel) throws AlertException {
-        ExistingIssueDetails<T> existingIssue = null;
-        ItemOperation searchResultOperation = ItemOperation.UPDATE;
+    private List<ActionableIssueSearchResult<T>> findIssueByProjectIssueModel(ProjectIssueModel projectIssueModel) throws AlertException {
+        List<ActionableIssueSearchResult<T>> searchResults = new LinkedList<>();
 
-        IssueTrackerSearchResult<T> searchResponse = exactIssueFinder.findExistingIssuesByProjectIssueModel(projectIssueModel);
+
+        IssueTrackerSearchResult<T> searchResponse = exactIssueFinder.findExistingIssuesByProjectIssueModel(projectIssueModel,DEFAULT_MAX_RESULTS);
```

Comment:
The usage of max results here is very specific so we don't introduce a performance hit from a completely unbound list of Jira issues. Perhaps we should include a comment stating why we have an upper limit of 100 in the event we need to make future changes or at least detail why this change was made.